app/blog/2017-10-18-upcoming-changes.md

```diff
@@ -0,0 +1,146 @@
+---
+title:  "Changes to shadow DOM v0 and HTML imports"
+---
+
+
+In the near future, Chrome is rolling out two changes that may affect existing Polymer 1.x and 2.x websites:
+
+
+*   Shadow-piercing selectors (`/deep/` and `::shadow`). Chrome 63 will disable the shadow-piercing selectors in style sheets. This change only affects sites using Polymer 1.x that are using native shadow DOM.
```

Comment:
Suggest

>Shadow-piercing selectors (`/deep/` and `::shadow`) will be disabled.

Instead of 

>Shadow-piercing selectors (`/deep/` and `::shadow`).